src/oracle/connection/stmt.rs

```diff
@@ -17,13 +17,20 @@ pub struct Statement {
     buffers: Vec<Box<[u8]>>,
     sizes: Vec<i32>,
     indicators: Vec<Box<ffi::OCIInd>>,
+    pub mysql: String,
```

Comment:
I do not think this needs to be public, prefer `pub(crate)` if it's used internally.